app/javascript/components/search/results/DataRepoStudy.js

```diff
@@ -0,0 +1,66 @@
+/* show a search result entry from Data Repo */
+import React from "react";
+export const descriptionCharacterLimit = 750
+export const summaryWordLimit = 150
+
+import {getDisplayNameForFacet} from "providers/SearchFacetProvider";
+
+export function shortenDescription(textDescription) {
+  const suffixTag = <span className="detail"> ...(continued)</span>
+  const displayedStudyDescription = { __html: textDescription.slice(0, descriptionCharacterLimit) }
+  if (textDescription.length>descriptionCharacterLimit) {
+    return <>
+      <span className="studyDescription" dangerouslySetInnerHTML={displayedStudyDescription}></span>{suffixTag}
+    </>
+  } else {
+    return <><span className = 'studyDescription' dangerouslySetInnerHTML={displayedStudyDescription}></span></>
```

Comment:
Blocker: SCP JS always uses kebab-case for attribute values.

```suggestion
      <span className="study-description" dangerouslySetInnerHTML={displayedStudyDescription}></span>{suffixTag}
    </>
  } else {
    return <><span className="study-description" dangerouslySetInnerHTML={displayedStudyDescription}></span></>
```

(It also typically double quotes them with no space around the equals sign.)

Reply:
Interesting... I actually have no idea how that got in there.  Will change it back.